scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -404,8 +399,7 @@ static void refresh_cpumasks(u32 layer_id)
 
 		if ((u8_ptr = MEMBER_VPTR(layers, [layer_id].cpus[cpu / 8]))) {
 			if (*u8_ptr & (1 << (cpu % 8))) {
-				if (refresh_layer_cpuc(cpuc, layer))
-					protected_changed = true;
+				protected_changed = refresh_layer_cpuc(cpuc, layer);
```

Comment:
This should be cumulative. You can do `|=` but `=` means that we're only accounting for the latest result.